app/src/main/java/com/sdp/movemeet/EditProfileActivity.java

```diff
@@ -67,6 +67,9 @@ protected void onCreate(Bundle savedInstanceState) {
             userId = user.getUid();
             storageReference = FirebaseStorage.getInstance().getReference();
             loadRegisteredUserProfilePicture(userId);
+        } else {
+            startActivity(new Intent(getApplicationContext(), LoginActivity.class)); // sending the user to the "Login" activity
+            finish();
```

Comment:
Yes, good redirection, this is important everywhere in fact, right!

Reply:
Yes, that the main point of my modifications for this week. On each page requires a connection to be accessed, I added this test. This way, if a user ends up on one of the pages requiring a connection when he is not connected, he will be redirected to the login page